introduction.md

```diff
@@ -6,5 +6,19 @@ I spend most of my time doing research and find a new way to do existing task wh
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?